### PR TITLE
Mayhem integration

### PR DESCRIPTION
**Pull request by code-terror**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,71 +1,20 @@
-# Builds images used by the bridge.
```
Comment (xansec) [inline Dockerfile]: Please do not modify upstream Dockerfiles.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+project: code-terror/parity-bridges-common
```
Comment (xansec) [inline Mayhemfile]: Please use a generic project name (omit your username).

File: ci.Dockerfile [hunk 3]
```diff
@@ -1,53 +0,0 @@
-# This file is a "runtime" part from a builder-pattern in Dockerfile, it's used in CI.
```
Comment (xansec) [inline ci.Dockerfile]: Please do not delete upstream files without good reason.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by code-terror** [comment]
Pushed the requested changes.. These were the initial submissions I worked on..So I was experimenting and playing around..So you would notice these issues..

**Comment by code-terror** [comment]
@xansec

**Review by xansec — CHANGES_REQUESTED** [review]
Hmm, the two upstream Dockerfiles are still showing as modified, but I can't figure out why? Maybe it's the missing newline? Please make the requested changes.

**Comment by code-terror** [comment]
@xansec should be done now

**Comment by xansec** [comment]
Awesome!

**Pull request merged**